GenericTypeType.Tests/GenericTypeType.Tests.csproj

```diff
@@ -9,28 +9,31 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <NeosDirectory Condition="'$(NeosDirectory)' == ''">C:\Program Files (x86)\Steam\steamapps\common\NeosVR</NeosDirectory>
-    <NeosCoreLibs>$(NeosDirectory)\Neos_Data\Managed</NeosCoreLibs>
-    <NeosNmlLibs>$(NeosDirectory)\nml_libs</NeosNmlLibs>
-    <NeosLibs>$(NeosDirectory)\Libraries</NeosLibs>
+	<ResonitePath>$(MSBuildThisFileDirectory)NeosVR</ResonitePath>
```

Comment:
you make my indentation sad.

Reply:
Oh geez. I'm sorry about that. It should've been

```
<PropertyGroup>
  <ResonitePath>...
```

I thought I made sure the indentation was good, but I missed this one for some reason.